### PR TITLE
[Preview] ISPN-10594 Remove WildFly server

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.jboss.org/browse/ISPN-10594

### CODE REVIEW

**Comment by ryanemerson** [comment]
"−48,171" :tada:

**Comment by tristantarrant** [comment]
I've probably broken the WF modules, but let's see

**Comment by tristantarrant** [comment]
CI looks good

File: wildfly/versions/pom.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-parent</artifactId>
+      <version>10.0.0-SNAPSHOT</version>
+      <relativePath>../../pom.xml</relativePath>
+   </parent>
+
+   <groupId>org.infinispan.wildfly</groupId>
+   <artifactId>infinispan-wildfly-versions</artifactId>
```
Comment (ryanemerson) [inline wildfly/versions/pom.xml]: Do we need a versions module any more? Or can we just add directly to `infinispan-wildfly`?

Comment (tristantarrant) [reply wildfly/versions/pom.xml]: I didn't want to introduce too many changes here.
There will also need to be a huge cleanup of the build-configuration/pom.xml to remove a ton of unneeded dependencies.

Comment (ryanemerson) [reply wildfly/versions/pom.xml]: Ok that's reasonable. I'm happy with the changes

**Pull request merged**